app/templates/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "screwdriver-<%= name %>",
+  "version": "1.0.0",
+  "description": "<%= description %>",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "jscs . && jshint .",
+    "test": "jenkins-mocha --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:screwdriver-cd/<%= name %>.git"
+  },
+  "homepage": "https://github.com/screwdriver-cd/<%= name %>",
+  "bugs": "https://github.com/screwdriver-cd/<%= name %>/issues",
+  "keywords": [
+    "screwdriver",
+    "yahoo"
+  ],
+  "license": "BSD-3-Clause",
+  "author": "<%= author %> <<%= email %>>",
+  "contributors": [
+    "Dao Lam <d2lam@yahoo-inc.com>",
+    "Darren Matsumoto <bobafett@yahoo-inc.com>",
```

Comment:
Preference is my github.com username plz 😺 
`Darren Matsumoto <aeneascorrupt@gmail.com>`